src/decoder.js

```diff
@@ -1,9 +1,19 @@
 import consensysDecode from 'abi-decoder';
+import { isAddress } from './web3/utils';
 
+const decodedAddressList = [];
 
-export const addABI = (abi) => {
+export const getDecodedAddresses = () => {
+  const returnedAddresses = decodedAddressList;
+  return returnedAddresses;
```

Comment:
you can do it just in one line `export const getDecodedAddresses = () => returnedAddresses `;